modules/common/src/main/scala/com/snowplowanalytics/snowplow/postgres/streaming/StreamSink.scala

```diff
@@ -26,4 +28,11 @@ object StreamSink {
         case c: StreamSinkConfig.Local => LocalSink.create(c, blocker)
       }
     }
+
+  def streamExists[F[_]: Async](config: Option[StreamSinkConfig])(implicit ec: ExecutionContext): F[Boolean] =
```

Comment:
Maybe this should be called something like `targetExists` to distinguish it from `KinesisSink.streamExists`?